src/libostree/ostree-bootloader-uboot.c

```diff
@@ -78,32 +78,32 @@ append_system_uenv (OstreeBootloaderUboot   *self,
   kargs = _ostree_kernel_args_from_string (bootargs);
   ostree_arg = _ostree_kernel_args_get_last_value (kargs, "ostree");
   if (!ostree_arg)
-  {
-    g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
-                         "No ostree= kernel argument found in boot loader configuration file");
-    return FALSE;
-  }
+    {
+      g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
+                           "No ostree= kernel argument found in boot loader configuration file");
+      return FALSE;
+    }
   ostree_arg += 1;
   uenv_path = glnx_strjoina (ostree_arg, "/usr/lib/ostree-boot/uEnv.txt");
   uenv_fd = openat (self->sysroot->sysroot_fd, uenv_path, O_CLOEXEC | O_RDONLY);
   if (uenv_fd != -1)
-  {
-    char *uenv = glnx_fd_readall_utf8 (uenv_fd, NULL, cancellable, error);
-    if (!uenv)
     {
-      g_prefix_error (error, "Reading %s: ", uenv_path);
-      return FALSE;
+      char *uenv = glnx_fd_readall_utf8 (uenv_fd, NULL, cancellable, error);
+      if (!uenv)
+        {
+          g_prefix_error (error, "Reading %s: ", uenv_path);
+          return FALSE;
+        }
+      g_ptr_array_add (new_lines, uenv);
     }
-    g_ptr_array_add (new_lines, uenv);
-  }
   else
-  {
-    if (errno != ENOENT)
     {
-      g_prefix_error (error, "openat %s: ", uenv_path);
-      return FALSE;
+      if (errno != ENOENT)
+        {
+          g_prefix_error (error, "openat %s: ", uenv_path);
```

Comment:
These two lines need to be `return glnx_throw_errno_prefix (error, "openat(%s)", uenv_path)`.

Reply:
OK, that wasn't a new issue in this patch.  I fixed it up though anyways.  Thanks for the work on this!